### PR TITLE
chore: add coverage

### PR DESCRIPTION
**Pull request by agonzalez0515**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,3 +7,5 @@ deploy:
   skip_cleanup: true
   on:
     branch: master
+after success:
```
Comment (gr2m) [inline .travis.yml]: it’s `after_success`

Comment (agonzalez0515) [reply .travis.yml]: oops! fixed!

File: .travis.yml [hunk 2]
```diff
@@ -7,3 +7,5 @@ deploy:
   skip_cleanup: true
   on:
     branch: master
+after_success:
+  - npm run converage
```
Comment (gr2m) [inline .travis.yml]:
```diff
- converage
+ coverage
```

:)

Comment (agonzalez0515) [reply .travis.yml]: lol goodness! I should just copy paste instead of typing it out.

**Comment by gr2m** [comment]
can you please add `.nyc_output` to `.gitignore`?

**Comment by gr2m** [comment]
oh and also add `coverage` sorry!

When you run `./node_modules/.bin/tap --coverage-report=lcov` you get an html coverage report that you can browse locally which is very helpful. This report is saved in the `coverage` folder so we ignore it in all Hoodie repos, too

**Pull request merged**